src/figvalidationservice/figcrpords.yaml

```diff
@@ -273,6 +274,40 @@ paths:
             type: object
             properties: {}
 definitions:
+  matchingApplicantsOrdsResponse:
+    type: object
+    required:
+    - found_party_id
+    - found_surname
+    - found_first_name
+    - found_second_name
+    - found_birth_date
+    - found_drivers_license
+    - found_birth_place
+    - found_gender
+    - status_code
+    - status_message
+    properties:
+      found_party_id:
+        type: string
```

Comment:
alphanumeric or integer?

Reply:
it's defined as a string pretty much everywhere else, but to me it's weird seeing an ID defined as a string.

moving on.

Reply:
It's an integer, i will talk to Carol about it, for now this is what is served by ords...

Reply:
that's cool. no need to follow-up if that's what you're getting. 💯